### PR TITLE
Add Docusaurus

### PR DESCRIPTION
**Pull request by relsteiner**
Follow up on https://github.com/practicajs/practica/issues/4 -

- Init Docusaurus in `/docusaurus` folder
- Add one example blog
- Add example intro doc
- Add decisions category with an example decision making doc
- Use github actions to deploy docs to github pages

### CODE REVIEW

File: .github/workflows/deploy-docusaurus.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'docusaurus/**'
```
Comment (goldbergyoni) [inline .github/workflows/deploy-docusaurus.yml:8]: @relsteiner Thought - Shall we maybe put this under ~root/docs/documentation-site so all our docs are centralized in a single folder?

File: .github/workflows/deploy-docusaurus.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci
+      - name: Build website
+        run: npm run build
+      - name: Deploy to GitHub Pages
```
Comment (goldbergyoni) [inline .github/workflows/deploy-docusaurus.yml:28]: Leme know to connect a domain to GH pages?

**Review by goldbergyoni — APPROVED** [review]
@relsteiner Great feature to our repo. Left a minor comment and leaving you with the judgment whether to fix or merge now.

Know how to connect domain name to the site?

File: .github/workflows/deploy-docusaurus.yml (hunk 1 above)
Comment (relsteiner) [reply .github/workflows/deploy-docusaurus.yml:8]: I thought that with time `/docuzaurus` will become our single documentation folder (already migrated the decisions folder from `/docs` to `/docuzaurus`).

Comment (goldbergyoni) [reply .github/workflows/deploy-docusaurus.yml:8]: @relsteiner Consider the following scenario - A developer is visiting the repo, not the website (many prefer GitHub vs custom site), she is looking to read some contribution/tech docs, she inspects the repo folders for a doc/docs/documentation folder but there is not any... She only sees this weird word 'docusaurus' and believe it contains something irrelevant

Not critical for now, down the road I would consider changing

**Pull request merged**